src/test/java/org/joda/beans/sample/SimpleJson.java

```diff
@@ -99,1564 +99,1618 @@ public final class SimpleJson implements ImmutableBean, Serializable {
     private final ImmutableMap<String, List<String>> listInMap;
     @PropertyDefinition
     private final ImmutableMap<String, List<Integer>> listNumericInMap;
+    @PropertyDefinition
+    private final ImmutableMap<String, Object> objectInMap;
+
+  //------------------------- AUTOGENERATED START -------------------------
```

Comment:
I believe this large diff must be due to using a later version of JodaBeans.

Reply:
The problem is that it is generated with 2 space indent when it needs to be 4 space. You'll need to generate manually or adjust the plugin settings.

Reply:
Ah yes hadn't noticed that